Products.cs

```diff
@@ -9,7 +9,7 @@ public class Products
 
         public override string ToString()
         {
-            return $"{nameof(this.Description)}: {this.Description}: {this.Description} {this.Price}";
+            return $"{nameof(this.Description)}: {this.Description}     {nameof(this.Price)} {this.Price}";
```

Comment:
Too many spaces.

Also at the beginning I saw the declaration of enums, this must to be like a class declaration and the name must to start with uppercase